### ISSUE TITLE
current HEAD does not build.

### ISSUE THREAD

**Issue by timobaumann** [fpecune/MUF-inMind-April2017-master#20]
Revision 9666980c517987512315fc643f5ff31e7f3b9208 (nor current HEAD) does not work for me. When I try to connet, I get: 

```
Exception in thread "session thread with id 54201342a4cfb96d" java.lang.NoSuchMethodError: edu.cmu.inmind.multiuser.sara.component.beat.BEAT.setCallback(Ledu/cmu/inmind/multiuser/sara/component/beat/BeatCallback;)V
	at edu.cmu.inmind.multiuser.sara.component.NLGComponent.<init>(NLGComponent.java:28)
	at edu.cmu.inmind.multiuser.sara.component.NLGComponent$$FastClassByGuice$$9a3b78f0.newInstance(<generated>)
	at com.google.inject.internal.DefaultConstructionProxyFactory$FastClassProxy.newInstance(DefaultConstructionProxyFactory.java:89)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.FactoryProxy.get(FactoryProxy.java:56)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:194)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
	at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1019)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1015)
	at com.google.inject.multibindings.Multibinder$RealMultibinder.get(Multibinder.java:375)
	at com.google.inject.multibindings.Multibinder$RealMultibinder.get(Multibinder.java:258)
	at com.google.inject.internal.ProviderInternalFactory.provision(ProviderInternalFactory.java:81)
	at com.google.inject.internal.InternalFactoryToInitializableAdapter.provision(InternalFactoryToInitializableAdapter.java:53)
	at com.google.inject.internal.ProviderInternalFactory.circularGet(ProviderInternalFactory.java:61)
	at com.google.inject.internal.InternalFactoryToInitializableAdapter.get(InternalFactoryToInitializableAdapter.java:45)
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54)
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:120)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1019)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1015)
	at com.google.inject.assistedinject.FactoryProvider2.invoke(FactoryProvider2.java:776)
	at com.sun.proxy.$Proxy10.create(Unknown Source)
	at edu.cmu.inmind.multiuser.controller.resources.DependencyManager.getOrchestrator(DependencyManager.java:43)
	at edu.cmu.inmind.multiuser.controller.session.Session.initialize(Session.java:90)
	at edu.cmu.inmind.multiuser.controller.session.Session.run(Session.java:103)
	at java.lang.Thread.run(Thread.java:748)
```
Steps to reproduce:
- do a clean clone of the project
- run ./gradlew build
- run java -jar build/libs/sara-with-dependencies-2.0.jar
- connect phone.

**Comment by sushma-ananda** [comment]
@timobaumann are you also using latest mobile android version: https://github.com/inmind-project/sara-integration.git ?? Because I am not getting above mentioned BEAT callback error when I followed :
your Steps to replicate the issue:

"do a clean clone of the project
run ./gradlew build
run java -jar build/libs/sara-with-dependencies-2.0.jar
connect phone."

**Comment by timobaumann** [comment]
no. I wasn't aware that I should update. 
However, no matter what phone is connected, that cannot have any effect on a NoSuchMethodError.

**Comment by sushma-ananda** [comment]
I have emailed you last week notifying you of changes and requesting a meeting. Can you please let me know on that?

**Comment by sushma-ananda** [comment]
This is fixed in 25816e518b1fad61682ca9a21a865ea8086b7df9
